low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -164,41 +161,37 @@ class SSPLDevDeploy:
             raise DeployError(rc, "%s - %s CMD: %s",
                 out, "Unable to remove cortx-prereq RPMS. Cleanup failed.", cmd)
+        print('Cleaning up local repos...')
+        for p in Path('/etc/yum.repos.d').glob('var_artifacts_*'):
+            p.unlink()
+        for p in Path('/etc/yum.repos.d').glob('cortx-storage.colo.seagate.com_*'):
+            p.unlink()
+        if os.path.exists('/etc/pip.conf'):
+            Path('/etc/pip.conf').unlink()
+        self._execute_cmd('yum clean all')
+        shutil.rmtree('/var/cache/yum/')
 
     def setup_yum_repos(self):
         """Setup common, 3rd_party and build specific repos."""
         if not (self.target_build or self.rpms_path):
             raise DeployError(1, "Target build or RPMS path is not given.")
         pkg_name = "cortx-prereq"
         print("INFO: INSTALLING %s..." % pkg_name)
-        prereq_url = ("http://cortx-storage.colo.seagate.com/releases/cortx/"
-            "third-party-deps/rpm/install-cortx-prereq.sh")
-        with urllib.request.urlopen(prereq_url) as response, open(
-                "install-cortx-prereq.sh", 'wb') as out_file:
-            shutil.copyfileobj(response, out_file)
-        os.chmod("install-cortx-prereq.sh", 0o775)
-        cmd = "sh install-cortx-prereq.sh"
-
-        if self.target_build:
-            cmd = cmd + " -b " + self.target_build
-            out, rc = self._execute_cmd(cmd)
-            if rc != 0:
-                raise DeployError(rc, "%s - %s CMD: %s",
-                    out, "Failed to install %s." % (pkg_name), cmd)
-            # Add repos to yum.repos.d
-            cmd1 = "yum-config-manager --add-repo %s/cortx_iso/" % self.target_build
-            cmd2 = "yum-config-manager --add-repo %s/3rd_party/" % self.target_build
-            for repo_cmd in [cmd1, cmd2]:
-                out, rc = self._execute_cmd(repo_cmd)
-                if rc != 0:
-                    raise DeployError(rc, "%s - %s CMD: %s",
-                        out, "Setup repo failed", repo_cmd)
-        else:
-            out, rc = self._execute_cmd(cmd)
-            if rc != 0:
-                raise DeployError(rc, "%s - %s CMD: %s",
-                    out, "Failed to install %s." % (pkg_name), cmd)
-            repo_cmd = "yum-config-manager --add-repo %s/3rd_party/" % self.build_url
+
+        repo_url = self.target_build if self.target_build else self.build_url
+
+        cmd1 = "yum-config-manager --add-repo %s/cortx_iso/" % repo_url
+        cmd2 = "yum-config-manager --add-repo %s/3rd_party/" % repo_url
+        with open('/etc/pip.conf', 'w') as pw:
+            cont = textwrap.dedent("""
+                [global]
+                timeout: 60
+                index-url: {}/python_deps/
+                trusted-host: localhost\n""").format(self.target_build)
+            pw.write(cont)
+        python_install_command = "yum install -y python3"
+        kafka_install_command = "yum install --nogpgcheck -y cortx-prereq"
```

Comment:
Why this variables is named as kafka_install_command?

Reply:
renamed.